epp-client/src/tests/de.rs

```diff
@@ -648,4 +649,15 @@ mod response {
         assert_eq!(ext.data.rgp_status.status, "pendingRestore".to_string());
         assert_eq!(object.tr_ids.server_tr_id, SVTRID.into());
     }
+
+    #[test]
```

Comment:
This commit won't pass tests because the commit implementing the code comes after this one, right? Maybe switch those around?

Reply:
Good catch, will switch